src/pygats/pygats.py

```diff
@@ -310,11 +311,18 @@ def press(ctx, key, n=1):
         pyautogui.press(key)
 
 
-def typewrite(ctx, message):
+def typewrite(ctx, message, lang='en'):
```

Comment:
'en' should be replaced with 'eng' due to Tesseract uses three letters to identify language